Formula/llvm.rb

```diff
@@ -272,6 +272,13 @@ def install
     # install llvm python bindings
     (lib/"python2.7/site-packages").install buildpath/"bindings/python/llvm"
     (lib/"python2.7/site-packages").install buildpath/"tools/clang/bindings/python/clang"
+
+    # Install symlinks for omp.h/libomp so they can be used in other formulae
+    mkdir lib/"libomp"
+    ["libgomp.dylib", "libiomp5.dylib", "libomp.dylib"].each { |libomp_dylib|
```

Comment:
Should `lib/"#{dylib}"` be replaced with `lib/dylib.to_s`, or does `dylib.to_s` need parentheses?